src/components/desktop/Page.vue

```diff
@@ -0,0 +1,84 @@
+<template>
```

Comment:
It looks like that this and HeaderDesktop are mostly the copy of corresponding mobile components, can we instead to make them look on desktop well to reduce code duplication?